packages/scandipwa/src/component/CheckoutDeliveryOptions/CheckoutDeliveryOptions.container.js

```diff
@@ -40,14 +46,20 @@ export class CheckoutDeliveryOptionsContainer extends PureComponent {
         } = props;
 
         const result = shippingMethods.find(
-            ({ method_code, carrier_code }) => `${carrier_code}_${method_code}` === shippingMethod
-        ) || shippingMethods[0] || {};
+            ({ method_code, carrier_code }) => (`${carrier_code}_${method_code}` === shippingMethod
+            ) && method_code !== STORE_IN_PICK_UP_METHOD_CODE
+        ) || (
+            shippingMethods[0] && shippingMethods[0].method_code !== STORE_IN_PICK_UP_METHOD_CODE
+                ? shippingMethods[0]
+                : {}
+        );
```

Comment:
Add some comments to this please, I do not understand what is going one here :)

Reply:
Added